.vscode/launch.json

```diff
@@ -23,6 +23,20 @@
           "--secure-port=8443", 
           "--runtime-config=testdata.datasource.grafana.app/v0alpha1=true"]
     },
+    {
+      "name": "Run API Server (query-localhost)",
```

Comment:
I have a bunch of configurations like these locally. I wonder if we should `gitignore` this file and add some tooling to generate them.

Reply:
Good question -- I think the value of simple debugging from vscode is worth keeping it out of gitignore.  When you say you have a bunch... should we add them?

Reply:
I have one for cloudinfo and then one for testdata and prometheus. We'd end up with one per core source, I don't know if that's too much noise or if there is a better solution for that